ucsmsdk/ucssession.py

```diff
@@ -549,21 +549,19 @@ def _logout(self):
         if self.__refresh_timer:
             self.__refresh_timer.cancel()
 
-        if self.__cookie:
```

Comment:
@vvb:  Any reason to remove this `if self.__cookie`. If we do login within try..catch, and we do logout in catch, and if login is unsuccessful then we should not fire `aaa_logout`.